app/controllers/packages_controller.rb

```diff
@@ -0,0 +1,19 @@
+class PackagesController < ApiController
+  def create
+    @package = Package.new(package_params)
+    @package.tracking_number = SequenceGeneratorService.generate
+
+    if @package.valid?
+      @package.save
+      render json: PackageSerializer.new(@package).serializable_hash, status: :created
+    else
+      render json: ErrorsSerializer.new(@package), status: :bad_request
+    end
+  end
```

Comment:
You can simplify it with 'if @package.save'. Cause save will run valid method anyway